src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj

```diff
@@ -6,6 +6,9 @@
     <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
     <IsTestProject>false</IsTestProject>
   </PropertyGroup>
+  <ItemGroup>
```

Comment:
Curious why this is the only file under "remove".

Reply:
Fixed in latest commit